src/components/v2/values/chartValuesDiff/ChartValuesView.tsx

```diff
@@ -1626,7 +1626,7 @@ const ChartValuesView = ({
             >
                 {renderValuesTabsContainer()}
                 <div className="chart-values-view__wrapper flexbox flex-grow-1 dc__overflow-hidden">
-                    <div className="chart-values-view__details dc__border-right dc__overflow-scroll">
+                    <div className="flexbox-col dc__gap-12 chart-values-view__details dc__border-right dc__overflow-scroll">
```

Comment:
Must be dc__gap-13

Reply:
this should be 12 only as we should prefer even values